ecommerce/views.py

```diff
@@ -114,6 +115,11 @@ def post(self, request, *args, **kwargs):
                     )
         else:
             order.status = Order.FULFILLED
+            try:
+                post_payment(order)
+            except FluidReviewException:
```

Comment:
I think we should make this an `except:` instead, just in case there are other exceptions coming from that function.

Reply:
✅